connector/connectionMetadata.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <connection-metadata>
-  <database enabled='true'>
-    <field />
+  <database enabled='true' label='Database'>
+    <field optional='true' />
```

Comment:
What happens if the user doesn't provide a value for the database? Don't many of the JDBC driver's API calls require a database to be set in order to work?

Reply:
That's a good point.  If the field isn't set, I see Tableau is able to connect but the user needs to choose the database from the drop-down menu.  In the logs I see Tableau making a  `MongoDatabaseMetaData` call to get the database names.  
I can change the database field to being required.